XamlControlsGallery/ControlPages/XamlUICommandPage.xaml

```diff
@@ -2,10 +2,8 @@
       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
       xmlns:local="using:AppUIBasics"
-      xmlns:local2="using:AppUIBasics.ControlPages"
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-      xmlns:muxcontrols="using:Microsoft.UI.Xaml.Controls"
```

Comment:
This isn't needed? Are the XamlUICommands below coming from WUX or MUX?

Reply:
Apparently from WUX, which surprised me. I first renamed it, than noticed that the reference was grayed out. Navigated to the page a couple of times after removing to make sure I didn't break anything :)